languagetool-language-modules/pt/src/main/resources/org/languagetool/rules/pt/grammar.xml

```diff
@@ -44227,25 +44227,335 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA.
+		  <example correction="por que">Não sei o motivo <marker>pq</marker> o plano não funcionou.</example>
+		  <example correction="por que">Gostaria de saber as razões <marker>pq</marker> recusou o convite.</example>
+	  </rule>
+	  <rule>
+        <antipattern>
+         <token>pq</token>
+         <token>?</token>
+         <example>Pq?</example>
+      </antipattern>
+      <pattern>
+         <token postag='_PUNCT'/>
+         <marker>
+         <token skip='-1'>pq</token>
+         </marker>
+	     <token>?</token>
+	  </pattern>
+      <message>Linguagem de Internet. Se "pq" estiver no início de uma pergunta, considere escrever <suggestion>por que</suggestion> por extenso.</message>
+		  <example correction="por que">Se não houve erro algum, <marker>pq</marker> não entregou o documento?</example>
+		  <example correction="por que">E faria tudo isso de novo, <marker>pq</marker> não?</example>
+		  <example correction="por que">O relatório foi entregue; <marker>pq</marker> não houve mudanças?</example>
+      </rule>
+      <rule>
+      <antipattern>
+         <token>pq</token>
+         <token>?</token>
+         <example>Pq?</example>
+      </antipattern>
+      <pattern>
+         <token postag='SENT_START'/>
+         <marker>
+         <token skip='-1'>pq</token>
+         </marker>
+	     <token>?</token>
+	  </pattern>
+      <message>Linguagem de Internet. Se "pq" estiver no início de uma pergunta, considere escrever <suggestion>por que</suggestion> por extenso.</message>
+		  <example correction="Por que"><marker>Pq</marker> não entregou o documento?</example>
+      </rule>
+      	  <rule>
+	  <antipattern>
+         <token>pq</token>
+         <token postag='_PUNCT'/>
+         <example>Ele não veio e eu nem sei pq.</example>
+	  </antipattern>
+	  <antipattern>
+	     <token>,</token>
+	     <marker>
+         <token skip='-1'>pq</token>
+         </marker>
+         <token>?</token>
+         <example>Ele não veio, pq?</example>
+      </antipattern>
+      <pattern>
+	     <token>,</token>
+	     <marker>
+         <token skip='-1'>pq</token>
+         </marker>
+         <token postag='_PUNCT'/>
+	  </pattern>
+      <message>Linguagem de Internet. Se "pq" expressar uma justificativa, considere escrever <suggestion>porque</suggestion> por extenso. Se puder substituir por "(razão) pela qual" ou se for uma pergunta, utilize "por que".</message>
+		  <example correction="porque">Ele não se importou, <marker>pq</marker> acabou voltando.</example>
+      </rule>
+      <rule>
+        <antipattern>
+         <token regexp='yes'>o|os|um|uns|algum|alguns|nenhum|nenhuns|esse|esses|desse|desses|este|estes|deste|destes|neste|nestes|nesse|nesses|aquele|aqueles|daquele|daqueles|do|dos</token>
+         <marker><token regexp='yes'>pq|pqs</token></marker>
+         <token regexp='yes'>d(?:a|e|o|isso|este|esta|ess[ea]|isto|aquilo|aquela|aquele)s?|n(?:isso|este|esta|ess[ea]|isto|aquilo|aquela|aquele)s?|[\.\?!;]</token>
```

Comment:
same here: like I said maybe it should be this way but I am not sure about the "?"